tool/modules/uds_fuzz.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/env python
```

Comment:
Shebang should not be included in CC modules, as they are run through `cc.py` rather than directly